src/main/controllers/admin/ManualUploadController.ts

```diff
@@ -15,11 +15,17 @@ export default class ManualUploadController {
         formCookie = req.cookies['formCookie'];
         const formData = formCookie ? JSON.parse(formCookie) : null;
 
+        let nonStrategicUpload = false;
```

Comment:
This could be simplified to: 

const nonStrategicUpload = req.query?.['non-strategic'] === 'true'

Reply:
done